osquery/tables/system/windows/services.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winsvc.h>
+#include <string>
+
+#include "osquery/tables/system/windows/registry.h"
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#pragma comment(lib, "Advapi32.lib")
+
+namespace osquery {
+namespace tables {
+
+const std::string kSvcStartType[] = {
+    "BOOT_START", "SYSTEM_START", "AUTO_START", "DEMAND_START", "DISABLED"};
+
+const std::string kSvcStatus[] = {"UNKNOWN",
+                                  "STOPPED",
+                                  "START_PENDING",
+                                  "STOP_PENDING",
+                                  "RUNNING",
+                                  "CONTINUE_PENDING",
+                                  "PAUSE_PENDING",
+                                  "PAUSED"};
+
+const std::map<int, std::string> kServiceType = {
+    {0x00000010, "OWN_PROCESS"},
+    {0x00000020, "SHARE_PROCESS"},
+    {0x00000100, "INTERACTIVE_PROCESS"},
+    {0x00000110, "OWN_PROCESS   (Interactive)"},
```

Comment:
It looks like you're placing this data into an SQL column, while the spacing looks nice in code, it will become annoying when slicing and dicing the SQL output. ;)